Astrolabe.lua

```diff
@@ -125,7 +125,7 @@ local issecurevariable = issecurevariable
 local real_GetCurrentMapAreaID = GetCurrentMapAreaID
 local function GetCurrentMapAreaID()
 	local id = real_GetCurrentMapAreaID();
-	if ( id < 0 and GetCurrentMapContinent() == WORLDMAP_WORLD_ID ) then
+	if ( id < 0 and GetCurrentMapContinent() == WORLDMAP_AZEROTH_ID ) then
```

Comment:
Yeah, WORLDMAP_WORLD_ID has gone the way of the dodo.
I wonder why more things have not broken with users of WORLDMAP_WORLD_ID ?